lib/task.js

```diff
@@ -43,12 +43,22 @@ class Task extends Subject {
 		this.title = task.title;
 		this.skip = task.skip || defaultSkipFn;
 		this.task = task.task;
+		this.taskOptions = task.options;
```

Comment:
This is moot. You can already access it as `this.task.options`.

Reply:
Okay i will `this.task.options`.